spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/resource/ResourceController.java

```diff
@@ -67,11 +68,19 @@ public ResourceController(ResourceRepository resourceRepository,
 		this.helper.setAlwaysUseFullPath(true);
 	}
 
-	@RequestMapping("/{name}/{profile}/{label}/**")
 	public String resolve(@PathVariable String name, @PathVariable String profile,
```

Comment:
Since it is a `RestController` I am not too concerned about backward compatibility.  If it is just unit tests that break, that is not a big deal.